test/common/DaemonConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
```

Comment:
Should project name/default namespace be "Microsoft.Azure.Devices.Edge.Test.Common"?